packages/lab/stories/form-field-next/form-field-next.stories.tsx

```diff
@@ -206,3 +206,38 @@ export const LongLabelsWithWrap: ComponentStory<typeof FormFieldNext> = (
     </FlowLayout>
   );
 };
+
+export const WithValidation: ComponentStory<typeof FormFieldNext> = (
+  props
+) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+    <FormFieldNext
+      validationStatus="error"
+      label="Form Field label"
+      helperText="Helper text"
+      {...props}
+    >
+      <InputNext validationStatus="error" defaultValue="Input value" />
+    </FormFieldNext>
+    <FormFieldNext
+      validationStatus="warning"
+      label="Form Field label"
+      helperText="Helper text"
+      {...props}
+    >
+      <InputNext validationStatus="warning" defaultValue="Input value" />
+    </FormFieldNext>
```

Comment:
@alycrys problem with current implementation: having to provide prop to both FF and Input
Would it be best to use CSS to add styling to Input if prop is given in FF? This would apply to checkbox etc too, so would be a lot of styling responsibility in FF

Reply:
I think adding it to the FF context will keep things consistent...until we get rid of the context (if/when).